### ISSUE TITLE
Operator MakeFile target throwing errors during binary installation.

### ISSUE THREAD

**Issue by rootxrishabh** [kubearmor/KubeArmor#1609]
## Bug Report

**To Reproduce**

Operator MakeFile uses a [go-get-tool](https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorOperator/Makefile#L95) function which downloads necessary binaries for certain targets/tasks but is absent. This results in make errors - 

   `make: Nothing to be done for 'controller-gen'.`

**Expected behavior**

Make should successfully install controller-gen binary. 

**Solution**

Creating a function to enable `go install` should solve these. For [ref](https://github.com/kubernetes-sigs/kubebuilder/blob/2ce2fe9a2c8ff0c756d8d39cd8fd7f872a466b9c/pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go#L266-L279)

**Comment by UtkarshUmre** [comment]
i would like to work on this

**Comment by rootxrishabh** [comment]
Hey @UtkarshUmre thanks for the interest. Looking forward to the PR. Let me know if you have any questions.

**Comment by UtkarshUmre** [comment]
yes @rootxrishabh I see it's labeled as a good first issue, and I'm excited to get started. Would you mind assigning it to me ?

**Comment by rootxrishabh** [comment]
Hey @UtkarshUmre any updates? Let me know if you need help with something.

**Comment by UtkarshUmre** [comment]
Hey @rootxrishabh  I haven't had the chance to address this issue yet due to ongoing exams and recent travel. However, I'll be starting on it tomorrow. I'll keep you updated if any issues arise. Thanks for checking in

**Comment by AllenGeorge08** [comment]
@UtkarshUmre Are you still working on this ..If not , I would like to do it
